src/odemis/driver/xt_client.py

```diff
@@ -1694,19 +1694,27 @@ def _set_scanner(self, scanner_name):
         return scanner_name
 
 
+# Very approximate values
+PRESSURE_VENTED = 100e3  # Pa
```

Comment:
Isn't it confusing to have a pressure value here? Especially since we also have a .pressure VA. I have the feeling that it would be more intuitive to just use an on/off or True/False value on this axis.

Reply:
Normally the vacuum axis linked to the pressure. It's sort of the "target" pressure. That's the idea behind it.